shanoir-ng-datasets/src/main/java/org/shanoir/ng/dataset/model/DatasetModalityType.java

```diff
@@ -38,7 +38,10 @@
 	PET_DATASET(5),
 
 	// EEG Dataset
-	EEG_DATASET(6);
+	EEG_DATASET(6),
+
+	// GENERIC Dataset
+	OTHER_DATASET(7);
```

Comment:
Hi Jean-Come, why should generic have the name other_dataset? Maybe it would be more easy to align the naming?